addons/MountMuzzle/data/settings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.1" ?>
```

Comment:
Please remove this file, it will be auto-generated. You should never check anything into the `data` sub-folder of an addon, as that is an entirely client-side managed folder.

Reply:
Done.